pvlib/soiling.py

```diff
@@ -69,15 +69,22 @@ def hsu(rainfall, cleaning_threshold, tilt, pm2_5, pm10,
 
     # never use mutable input arguments
     if depo_veloc is None:
-        depo_veloc = {'2_5': 0.004, '10': 0.0009}
+        depo_veloc = {'2_5': 0.0009, '10': 0.004}
 
     # accumulate rainfall into periods for comparison with threshold
     accum_rain = rainfall.rolling(rain_accum_period, closed='right').sum()
     # cleaning is True for intervals with rainfall greater than threshold
     cleaning_times = accum_rain.index[accum_rain >= cleaning_threshold]
+    
+    # determine the time intervals in seconds (dt_sec)
+    dt = rainfall.index.values       # datetimes in nanoseconds
+    dt2 = np.roll(dt.copy(),1)       # shift array values by one place
+    dt3 = np.delete((dt-dt2)/1e9,0)  # subtract shifted values from original and convert to seconds
+    # append the last value to the end so there are the same number of elements in the array
+    dt_sec = np.append(dt3,dt3[-1]).astype('float64') 
 
     horiz_mass_rate = pm2_5 * depo_veloc['2_5']\
-        + np.maximum(pm10 - pm2_5, 0.) * depo_veloc['10']
+        + np.maximum(pm10 - pm2_5, 0.) * depo_veloc['10'] * dt_sec
```

Comment:
I think this still needs a set of parentheses.  Right now it is `v10 C10 + v2.5 C2.5 * t`, but it should be `(v10 C10 + v2.5 C2.5) * t`, no?

Reply:
Thanks, I agree with adding the parentheses.

Reply:
I will need to change the tests to account for this change.

Reply:
I just added a new test for variable time intervals.